complete_test.go

```
@@ -0,0 +1,188 @@
+	"bufio"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/posener/complete"
+)
+
+// testFlagPredictor predicts values for the "--color" flag.
+type testFlagPredictor struct{}
```

golangci-lint (2.12.2)

[error] type testFlagPredictor is unused

(unused)